generators/client/index.js

```diff
@@ -541,168 +541,9 @@ module.exports = JhipsterClientGenerator.extend({
-                'app/services/user/user.service.js'
-            ];
-            if (this.enableTranslation) {
-                appScripts = appScripts.concat([
-                    'app/blocks/handlers/translation.handler.js',
-                    'app/blocks/config/translation.config.js',
-                    'app/blocks/config/translation-storage.provider.js',
-                    'app/components/language/language.service.js',
-                    'app/components/language/language.constants.js',
-                    'app/components/language/language.filter.js',
-                    'app/components/language/language.controller.js',
-                    'app/layouts/navbar/active-menu.directive.js'
-                ]);
-            }
-            if (this.enableSocialSignIn) {
-                appScripts = appScripts.concat([
-                    'app/account/social/directive/social.directive.js',
-                    'app/account/social/social-register.state.js',
-                    'app/account/social/social-register.controller.js',
-                    'app/account/social/social.service.js'
-                ]);
-            }
-            if (this.authenticationType == 'jwt') {
-                appScripts = appScripts.concat([
-                    'app/services/auth/auth.jwt.service.js',
-                    'app/blocks/interceptor/auth.interceptor.js'
-                ]);
-            }
-
-            if (this.authenticationType == 'oauth2') {
-                appScripts = appScripts.concat([
-                    'app/services/auth/auth.oauth2.service.js',
-                    'app/blocks/interceptor/auth.interceptor.js'
-                ]);
-            }
-
-            if (this.authenticationType == 'session') {
-                appScripts = appScripts.concat([
-                    'app/services/auth/sessions.service.js',
-                    'app/services/auth/auth.session.service.js',
-                    'app/account/sessions/sessions.state.js',
-                    'app/account/sessions/sessions.controller.js'
-                ]);
-            }
-
-            if (this.websocket == 'spring-websocket') {
-                appScripts = appScripts.concat([
-                    'app/admin/tracker/tracker.state.js',
-                    'app/admin/tracker/tracker.controller.js',
-                    'app/admin/tracker/tracker.service.js'
-                ]);
-            }
-
-            if (this.applicationType == 'gateway') {
-                appScripts = appScripts.concat([
-                    'app/admin/gateway/gateway.state.js',
-                    'app/admin/gateway/gateway.controller.js',
-                    'app/admin/gateway/gateway.routes.service.js'
-                ]);
-            }
-
-            indexFile = html.appendScripts(indexFile, 'app/app.js', appScripts, {});
+            indexFile = html.append(indexFile, 'body', '\n<!-- build:js app/app.js  -->\n<!-- inject:js -->\n<!-- endinject -->\n<!-- endbuild -->\n');
```

Comment:
we could add this placeholder in the index.html template directly and avoid touching the file here